functions_framework_builder/test/builder_test.dart

```diff
@@ -62,39 +195,19 @@ const _ignoredLogMessages = {
       'SDK version.',
 };
 
-Map<String, String> _createPackageStub(
-  String testLibContent,
-) =>
-    {'$_pkgName|lib/test_lib.dart': testLibContent};
-
-// Ensure every test gets its own unique package name
-String get _pkgName => 'pkg$_pkgCacheCount';
-int _pkgCacheCount = 1;
-
-const _testLibContent = r'''
-import 'package:functions_framework/functions_framework.dart';
-import 'package:shelf/shelf.dart';
-
-@CloudFunction()
-Response handleGet(Request request) => Response.ok('Hello, World!');
-''';
-
-const _testGenPartContent = r'''
+String get _outputHeader => '''
 // GENERATED CODE - DO NOT MODIFY BY HAND
-// Copyright (c) 2020, the Dart project authors.
```

Comment:
See previous comment about retaining copyright even for generated code.

Reply:
ack

Reply:
ack